### ISSUE TITLE
Troubleshooting running `echo` command on Windows

### ISSUE THREAD

**Issue by AntoineKM** [Nutlope/aicommits#67]
Hi, I am making this issue to guide people who encounter the same problem as me on Windows.

Be careful while following the ``README.md`` command:
```bash
echo "OPENAI_KEY=<your token>" >> ~/.aicommits
```
This one might not support UTF-8 depending on your Windows configuration.

I did some testing to understand what the result was:
**Input**
![code](https://user-images.githubusercontent.com/54948363/219484349-a2713ef2-4983-4ff8-9e6d-c7d8c5cbf67a.png)

**Output**
![image](https://user-images.githubusercontent.com/54948363/219483833-bc80d09d-27e5-4c70-ab9e-a8be77c8d310.png)

And I quickly realized that the parse would not work.

Here are the only 3 solutions I have found to fix this problem:
1. Using `replace()`, but this solution doesn't seem ideal to me.
```js
const configString = (await fs.readFile(configPath, 'utf8')).replace(/[^\u0020-\u007E]/g, '');
```
2. Using environment variables #43 
3. Creating the file manually with vscode or notepad in UTF-8

**Comment by Nutlope** [comment]
Thanks for this issue! Maybe we should add the environment variable workaround as a footnote in the README for folks who run into issues on Windows.

**Comment by privatenumber** [comment]
I'll follow up with a quick `config` command. Better to set a secure example.

**Comment by privatenumber** [comment]
Closed via #71
